src/server.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react'
-import { flush } from './style'
 
+// fake flush
+function flush() {
+  return []
+}
+
+/* @deprecated */
```

Comment:
I would fail here or warn otherwise it will "fail" silently i.e. `flushToReact` and `flushToHTML` continue to work but output nothing